index.html

```diff
@@ -375,6 +377,12 @@ <h1>Unitary Fund</h1>
 
     </p>
 
+    <p class="subtitle"><a name="news">News</a></p>
+
+    <p>
+        <a href="news.html">Team intro</a>
```

Comment:
I missed to change this. it should be 
```
<a href="blog.html">Team intro</a>
```

Reply:
sorry I mean cut this all as we have no news.

Reply:
removed